packages/dds/legacy-dds/src/array/sharedArray.ts

```diff
@@ -224,6 +242,8 @@ export class SharedArrayClass<T extends SerializableTypeForSharedArray>
 		const entry = this.sharedArray[indexInternal];
 		assert(entry !== undefined, 0xb90 /* Invalid index */);
 		const entryId = entry.entryId;
+		// this.insertEntryId.delete(entryId);
```

Comment:
toggle should be able to recover deleted items so don't deleting the entryId here. I'll remove the comment later